### PR TITLE
Survey Modal A11y

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
**Description:**
- Adds a close button to survey modal
- Modal is now dismissed on esc
- Duplicate `role="dialog"` removed

**PR Checklist:**
- [x] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [x] Are all languages supported (Internationalization/Localization)?
- [x] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM!

File: lib/components/app/popup.tsx [hunk 1]
```diff
@@ -3,8 +3,12 @@ import { useIntl } from 'react-intl'
 import coreUtils from '@opentripplanner/core-utils'
 import React, { useEffect } from 'react'
 
+import { Icon } from '../util/styledIcon'
+import { X } from '@styled-icons/bootstrap/X'
```
Comment (binh-dam-ibigroup) [inline lib/components/app/popup.tsx:7]: Just a side comment that the "Close" icon in the app menu is just CSS.

**Comment by amy-corson-ibigroup** [comment]
Just an update, I added an `aria-label` to the `role="dialog"` container per some client feedback

File: lib/components/app/popup.tsx [hunk 2]
```diff
@@ -49,7 +75,21 @@ const PopupWrapper = ({ content, hideModal }: Props): JSX.Element | null => {
   const title = intl.formatMessage({ id: `config.popups.${id}` })
 
   return (
-    <Modal dialogClassName="fullscreen-modal" onHide={hideModal} show={shown}>
+    <Modal
+      aria-label={title}
+      dialogClassName="fullscreen-modal"
+      onEscapeKeyDown={hideModal}
```
Comment (miles-grant-ibigroup) [inline lib/components/app/popup.tsx:81]: Well done finding this!

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Looks and works great thanks for the changes! Just one opportunity to avoid re-adding the same feature twice

File: lib/components/app/popup.tsx [hunk 3]
```diff
@@ -15,6 +19,26 @@ type Props = {
   hideModal: () => void
 }
 
+const CloseModalButton = styled.button`
```
[the remark this answers is not in the document]
Comment (amy-corson-ibigroup) [reply lib/components/app/popup.tsx:22]: Good idea! Added in [8f0078f](https://github.com/opentripplanner/otp-react-redux/pull/896/commits/8f0078fb1583ef560ff2796f9644ecbcdf9d7342)

[the remark this answers is not in the document]
Comment (miles-grant-ibigroup) [reply lib/components/app/popup.tsx:22]: As discussed offline, we should re-do all of the x buttons at some point in the future. Glad they share some code now!

**Review by miles-grant-ibigroup — APPROVED** [review]
Looking grand

**Pull request merged**